### ISSUE TITLE
requests (gamepad support, crisp fullscreen and dark background option)

### ISSUE THREAD

**Issue by TheJoshCalibur** [taisel/GameBoy-Online#1]
i know already that gamepad api is here so i would lke that. also the fullscreen "pixels" are a bit blurry. and also a dark background with css sould be easy. otherwise this emulator is quite faster than googulator so keep up the good work!

**Comment by taisel** [comment]
Yeah, adding features onto this aging code is a quest into insanity. +1s for actual fixes should in large be for the rewrite I never got around to working on properly, even though I promised such. That being said, I agree interface cleanups are in order.

**Comment by test6j67** [comment]
Not an Issue, do not send issues, when really  there is not issue.
